core/controllers/blog_admin.py

```diff
@@ -23,17 +23,29 @@
 from core.controllers import base
 from core.controllers import domain_objects_validator as validation_method
 from core.domain import blog_services
-from core.domain import config_domain
-from core.domain import config_services
+from core.domain import platform_feature_services
+from core.domain import platform_parameter_domain
+from core.domain import platform_parameter_list
+from core.domain import platform_parameter_registry
 from core.domain import role_services
 from core.domain import user_services
 
-from typing import Dict, Final, List, Optional, TypedDict, Union
+from typing import Dict, Final, Optional, TypedDict
 
 BLOG_POST_EDITOR: Final = feconf.ROLE_ID_BLOG_POST_EDITOR
 BLOG_ADMIN: Final = feconf.ROLE_ID_BLOG_ADMIN
 
 
+class PlatformParamBlogAdminSchemaDict(TypedDict):
```

Comment:
Why does this need a separate type from the other platform parameters in the app?